### ISSUE TITLE
Opentoonz freezes/crashes when saving and loading

### ISSUE THREAD

**Issue by SmithyGCN** [opentoonz/opentoonz#3419]
**Describe the bug**
My version of Opentoonz (which has been updated to the latest version, thank you) has stopped saving and loading scenes preferring to crash when I do so. This has gotten very aggravating and I've tried all the usual fixes.

**To Reproduce**
List steps to reproduce the behavior.
1.Open program.
2.Try to save or load any scene.
3.

**Expected behavior**
To be able to open and continue animating as usual.

**Screenshots**
You don't need a screenshot to know what I'm talking about. Sorry for my rudeness.

**Device Information:**
 - OS: [e.g. Windows]
 - OpenToonz Version: [e.g. Nightly (2020-01-10)]
 - GPU: [e.g. NVIDIA RTX 2080]
 - GPU Driver Version: [e.g. 440.00]

**Comment by RodneyBaker** [comment]
> To Reproduce
> 1.Open program.
> 2.Try to save or load any scene.

Perhaps there is some additional step (or some files) we can examine as those two steps work as expected for the vast majority of users.


Without knowing more about your specific issue, I will recommend renaming your stuff directory (for example change it from "Opentoonz stuff" to "Opentoonz stuff (20200729)" and then reinstall to get a fresh stuff directory.  Then you can pull your previous scenes from your old stuff directory into the new as needed.

Is there a particular reason you are using a release dated 2020-01-10 and not a more current nightly release?
There have been many updates/changes since January.

LInk to most recent releases:  https://github.com/opentoonz/opentoonz_nightlies/releases

**Comment by SmithyGCN** [comment]
Thanks, but I don't know how to use the package file nor do I know how to update OpenToonz. ^^; Is it in the help file?

**Comment by RodneyBaker** [comment]
The package file is the release for the MacOS.
The Opentoonz.exe file is the one you'll want as a Windows user.

**Comment by SmithyGCN** [comment]
Thanks, but now it says I can't open the file due to permission not being granted. I've fiddled with the permissions and even ran as administrator, but I keep getting this same error.

![image](https://user-images.githubusercontent.com/68975222/88896327-50ca5080-d20f-11ea-8340-5c449840b089.png)

**Comment by jimboyeah** [comment]
It so troubled

**Comment by RodneyBaker** [comment]
> 

Antivirus programs tend to be a bit over protective.
Adjust the settings of Avast Antivirus (or perhaps better yet... get rid of it!) and you will likely see better results.  :)

**Comment by Triangle4** [comment]
Yeah having a lot of crashes on a top of the line system with top tier drives. I have 16tb mp600s on this system and 32 cores with 128gb of ram and still crashes / freezes when performing file operations. Yikes...

**Comment by andeon** [comment]
@RodneyBaker This reported issue technically does not provide useful information to identify the problem, so I recommend closing it. The user could test the nightly version and, if the issue persists, open a new report, since this concerns an old OpenToonz version.
